index.html

```diff
@@ -0,0 +1,198 @@
+              <a href="#"><i class="fab fa-twitter"></i></a>
+            </li>
+            <li>
+              <a href="#" class="my-page"><p>MyPage</p></a>
+            </li>
+            <li>
+              <a href="#" class="logout log-pad"><p>Logout</p></a>
+            </li>
+          </ul>
+        </nav>
+        <nav class="secondary-nav">
+          <div class="banner secondary-nav">
+            <a href="index.html">
+              <img class="logo-img" src="images/eagle-logo.jpeg" alt="logo" />
+              <p class="comp-name">Haddis Technology</p>
+            </a>
+          </div>
+          <ul class="secondary-nav nav-menu menu-nav">
+            <li class="nav-item">
+              <a href="index.html" class="link-nav home-link">Home</a>
+            </li>
+            <li class="nav-item">
+              <a href="about.html" class="link-nav">About</a>
+            </li>
+            <li class="nav-item"><a href="" class="link-nav">Program</a></li>
+            <li class="nav-item"><a href="" class="link-nav">Join</a></li>
+            <li class="nav-item"><a href="" class="link-nav">Sponsor</a></li>
+            <li class="nav-item"><a href="" class="link-nav">News</a></li>
+            <li class="nav-item">
+              <a href="" class="confference">Confference</a>
+            </li>
+          </ul>
+        </nav>
+        <div class="hamburger" id="hamburger">
+          <span class="l-1 bar"></span>
+          <span class="l-2 bar"></span>
+          <span class="l-3 bar"></span>
+        </div>
+      </div>
+      <section class="headlines">
+        <div class="advert-txt">
+          <p>"The software of your thoughts"</p>
+          <h1>WE COME WITH SMART TECHNO PRODUCTS!</h1>
+        </div>
+        <div class="box-txt">
+          <p>
+            In simple words, technology is when we take the concepts of science
+            and transform them into gadgets and devices that are useful to us as
+            human beings. When we look around us, almost everything is a result
+            of or a form of technology.
+          </p>
+        </div>
+        <div class="address-txt">
+          <p><strong>Next Zoom conference(monday july 25/2022)</strong></p>
+          <p>@ BahirDar, Ethiopia</p>
+        </div>
+      </section>
+
+      <!-- Main Program -->
+
+      <section class="main-program">
+        <h2 class="main-pro">See The All Program</h2>
+        <div class="card-container">
+          <div class="card">
+            <div class="card-img">
+              <i class="fa fa-users fa-2x" aria-hidden="true"></i>
+            </div>
+            <div class="card-title">Coding</div>
+            <div class="card-txt">
+              Lorem ipsum dolor sit, amet consectetur adipisicing elit. Tenetur,
+              minima non doloribus harum corrupti dolores nihil illo voluptates,
+              qui culpa quas voluptatem? Itaque, corrupti. Obcaecati.
+            </div>
+          </div>
```

Comment:
- [ ] Kindly ensure that you use `article` for each card instead of `div` to improve the semantic structure of your website